docs/fastdds/api_reference/dds_pim/xtypes/exception.rst

```diff
@@ -0,0 +1,10 @@
+.. _api_pim_xtypes_exception:
+
+.. rst-class:: api-ref
+
+InvalidArgumentError
```

Comment:
I have talked with Edu and he also prefers to keep the include folder structure in the API reference. It could be moved in a follow-up PR with the corresponding move of the header file